tests/test_model_saving.py

```diff
@@ -859,5 +859,46 @@ def test_sequential_mxnet_model_saving_no_compile():
         save_mxnet_model(model, prefix='test', epoch=0)
 
 
+@pytest.mark.skipif((K.backend() != 'mxnet'),
+                    reason='Supported for MXNet backend only.')
+@keras_test
+def test_sequential_get_mxnet_model_info():
```

Comment:
Shall we also test Functional Model Saving? using example model from test_functional_model_saving()

Reply:
Thanks. Added a test for functional model.